comp/core/workloadmeta/collectors/internal/kubeapiserver/deployments.go

```diff
@@ -132,7 +141,7 @@ func (p deploymentParser) Parse(obj interface{}) []workloadmeta.Entity {
 			ID:   string(util.GenerateKubeMetadataEntityID("apps", "deployments", deployment.Namespace, deployment.Name)),
 		},
 		EntityMeta: deploymentEntity.EntityMeta,
-		GVR:        deployment.GroupVersionKind().GroupVersion().WithResource("deployments"),
```

Comment:
For some reason, this was not populating the Group and Version, probably an issue with the kubernetes client parsing. With this PR, the issue shouldn't exist anymore.